src/store/hooks.ts

```diff
@@ -0,0 +1,11 @@
+import {
+  TypedUseSelectorHook,
+  useDispatch,
+  useSelector,
+} from 'react-redux';
+
+import type { RootState, AppDispatch } from './index';
+
```

Comment:
use me! (useAppDispatch/useAppSelector)
also, it looks like there's a typo in useAppDis**t**patch()

Reply:
but I suppose you could use it as is as long as you're aware there's a an extra t in there